### DISCUSSION TITLE
История событий

### DISCUSSION THREAD

**Discussion by fishenebelny** [TionAPI/HA-tion#80] [Ideas]
Может быть имеет смысла добавить историю не только переключения состояния с включено/выключено, но и изменения скорости вентилятора?
Сейчас получается температуры все записываются, состояния записываются, а скорость вентилятора нет.

**Comment by IATkachenko** [comment]
Чуть-чуть терпения.
В #79 добавлется fan-компонент, который обеспечит, в том числе, и этот функционал.

**Comment by IATkachenko** [comment]
Добавился, но похоже работает не совсем так, как надо: HA считает что fan -- это нечто что может быть включено или выключено, и относится к этому именно так...
Добавить сенсор?

**Reply by fishenebelny** [reply]
Если не сллжно. Так то в принципе можно и самому сделать этот сенсор при помощи шаблона.

**Reply by IATkachenko** [reply]
В интеграции это 3,5 строчки кода :)

**Comment by IATkachenko** [comment]
[v2.3.1](https://github.com/TionAPI/HA-tion/releases/tag/v2.3.1) делает все как надо, по аналогии с температурой.

**Comment by fishenebelny** [comment]
Игорь,

Спасибо за сенсор. Все супер. Но нет предела совершенству ) Сейчас если Тион выключен, то датчик скорости показывает скорость вентилятора, которая была на момент выключения. На мой взгляд в идеале надо, чтобы был 0. Это только для стремления к совершенству. В остальном все идеально работает.

**Reply by IATkachenko** [reply]
Вы знаете что делать: new issue plz.
Очень не удобно потом искать, когда все в кучу намешано.

**Reply by fishenebelny** [reply]
Ok ) Спасибо!
